realm/realm-library/src/main/java/io/realm/DynamicRealm.java

```diff
@@ -279,6 +279,22 @@ static DynamicRealm createInstance(RealmConfiguration configuration) {
     }
 
     /**
+     * Returns a distinct set of objects from a specific class that fulfill the multiple fields
+     * condition. If the result is sorted, the first object of a column will be returned in case of
+     * multiple occurrences, otherwise it is undefined which object is returned.
+     *
+     * @param className the Class to get objects of.
+     * @param fieldNames the multiple fields.
+     * @return a non-null {@link RealmResults} containing the distinct objects.
+     * @throws IllegalArgumentException if fieldNames is empty, or a field is null, does not exist,
+     * is an unsupported type, or points to a linked field.
+     */
+    public RealmResults<DynamicRealmObject> distinct(String className, String... fieldNames) {
```

Comment:
Wrong way to specify var ags with 1 and more arguments: http://jtechies.blogspot.dk/2012/07/item-42-use-varargs-judiciously.html